telegram/ext/_dispatcher.py

```diff
@@ -631,6 +672,34 @@ def remove_handler(self, handler: Handler, group: int = DEFAULT_GROUP) -> None:
             if not self.handlers[group]:
                 del self.handlers[group]
 
+    def drop_chat_data(self, chat_id: int) -> None:
+        """Used for deleting a key from the :attr:`chat_data`.
+
+        .. versionadded:: 14.0
+
+        Args:
+            chat_id (:obj:`int`): The chat id to delete from the persistence. The entry
+                will be deleted even if it is not empty.
+        """
+        del self._chat_data[chat_id]
```

Comment:
this will raise a `KeyError` if `chat_id` is not present - thought we decided on not doing that 🤔  I'd suggest `.pop(chat_id, None)` instead

Reply:
right